docs/static/winlogbeat-modules.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[winlogbeat-modules]]
+
+== Working with {winlogbeat} Modules
```

Comment:
You need a uniquer ID on this section so that the ID doesn't change if the topic title changes.